insights/combiners/httpd_conf.py

```diff
@@ -105,6 +57,8 @@ class HttpdConfAll(object):
     ConfigData = namedtuple('ConfigData', ['file_name', 'file_path', 'full_data_dict'])
 
     def __init__(self, httpd_conf):
+        deprecated(HttpdConfAll, "Import HttpdConfTree instead.")
```

Comment:
@xiangce since there is no way to see this deprecation in the documentation, would you mind to also add a note to the class docstring that this is deprecated?

Reply:
Thanks, @bfahr, I added the note in the latest commit.